Source/WebCore/page/ResizeObserver.h

```diff
@@ -49,14 +49,25 @@ struct ResizeObserverData {
     Vector<WeakPtr<ResizeObserver>> observers;
 };
 
+class NativeResizeObserverCallback : public RefCounted<NativeResizeObserverCallback> {
+public:
+    NativeResizeObserverCallback() = default;
+    virtual ~NativeResizeObserverCallback() = default;
+    virtual void onResize(const Vector<Ref<ResizeObserverEntry>>&) = 0;
+};
```

Comment:
Done, added `ResizeObserver*`, so that in the callback we could call `observer->unobserve(*target)` instead:
```
using NativeResizeObserverCallback = void (*)(const Vector<Ref<ResizeObserverEntry>>&, ResizeObserver*);
using JSOrNativeResizeObserverCallback = std::variant<RefPtr<ResizeObserverCallback>, NativeResizeObserverCallback>;
```